### PR TITLE
Support SpongeAPI 7.1.0

### PR DESCRIPTION
**Pull request by LXGaming**
### Changes
Updated Prism dependencies
Fixed Build issues

### CODE REVIEW

File: src/main/java/com/helion3/prism/api/records/BlockResult.java [hunk 1]
```diff
@@ -36,6 +36,7 @@
 import com.helion3.prism.util.BlockUtil;
 import com.helion3.prism.util.DataQueries;
 import org.spongepowered.api.world.BlockChangeFlag;
```
Comment (viveleroi) [inline src/main/java/com/helion3/prism/api/records/BlockResult.java]: I think this import is no longer used and can be removed?

Same is true for the UndoCommand.java, but I can't comment on that line.

**Review by viveleroi — CHANGES_REQUESTED** [review]

File: src/main/java/com/helion3/prism/api/records/BlockResult.java (hunk 1 above)
Comment (LXGaming) [reply src/main/java/com/helion3/prism/api/records/BlockResult.java]: Done

File: src/main/java/com/helion3/prism/api/records/PrismRecord.java [hunk 2]
```diff
@@ -301,30 +311,19 @@ private void writeEntity(Entity entity) {
...
+         * @param transaction
          */
-        private void writeItem(Item item) {
-            checkNotNull(item);
-
-            DataContainer itemData = item.toContainer();
-
-            // Because item actions are not currently actionable, copy only what we need
-            Optional<DataView> position = itemData.getView(DataQueries.Position);
-            if (position.isPresent()) {
```
Comment (viveleroi) [inline src/main/java/com/helion3/prism/api/records/PrismRecord.java:313]: Can you clarify why you dropped location data? That seems essential for items.

Comment (LXGaming) [reply src/main/java/com/helion3/prism/api/records/PrismRecord.java:313]: The location is set inside the `dropped` and `pickedUp` methods, The reason behind this is the Transaction doesn't have Location information instead I'm grabbing it from the source.

**Pull request merged**